src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -257,4 +280,150 @@ private static void writeFile(String writable) {
             DriverStation.reportWarning("Unable to save fluid constants to file.", false);
         }
     }
+
+    /**
+     * Xbox controller binding information.
+     * Contains the link between the Xbox's buttons' port and the NetworkTables key used to describe the action.
+     */
+    public enum XboxValue {
```

Comment:
This has been moved to its own class, and should be removed.

Reply:
@eandr127 I just tried removing it from Config.java but I got some other compilation error. It is probably safest if you do this one.

Reply:
I've removed the extra XboxValue enum